src/operator/operator.go

```diff
@@ -0,0 +1,746 @@
+package operator
+
+import (
+	"fmt"
+	"github.com/gojinja/gojinja/src/utils/slices"
+	"math"
+	"reflect"
+	"strings"
+)
+
+func ToInt(v any) (i int64, ok bool) {
```

Comment:
I'm confused; isn't it a copy&paste from #19 ?

Reply:
Other methods too

Reply:
It is

Reply:
Is that intentional? Will be deduplicated in the future?

Reply:
I unified the code in newest Commit